setup.py

```diff
@@ -45,8 +46,6 @@ def read(*rnames):
         'Intended Audience :: Developers',
         'License :: OSI Approved :: Zope Public License',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
```

Comment:
Adding `python_requires=">=3.7"` would be greatly appreciated by users who are about to get dropped.

Reply:
+1